engine/src/physics/plugin.cpp

```diff
@@ -81,16 +143,65 @@ void cubos::engine::physicsPlugin(Cubos& cubos)
                 auto force = Force{};
                 force.add(bundle.force);
 
+                auto torque = Torque{};
+                torque.add(bundle.torque);
+
                 auto impulse = Impulse{};
                 impulse.add(bundle.impulse);
 
+                auto angularImpulse = AngularImpulse{};
+                angularImpulse.add(bundle.angularImpulse);
+
                 cmds.add(ent, Mass{.mass = bundle.mass, .inverseMass = 1.0F / bundle.mass});
+                if (bundle.inertiaTensor != glm::mat3(0.0F))
+                {
+                    cmds.add(ent, Inertia{.inertia = bundle.inertiaTensor,
+                                          .inverseInertia = glm::inverse(bundle.inertiaTensor),
+                                          .autoUpdate = false});
+                }
                 cmds.add(ent, Velocity{.vec = bundle.velocity});
+                cmds.add(ent, AngularVelocity{.vec = bundle.angularVelocity});
                 cmds.add(ent, force);
+                cmds.add(ent, torque);
                 cmds.add(ent, impulse);
+                cmds.add(ent, angularImpulse);
                 cmds.add(ent, AccumulatedCorrection{});
             }
         });
 
+    // Should this be here?
```

Comment:
Good question. I think you can keep it here for now, but maybe in the future we should keep those in a separate sub plugin?

Reply:
I think so yeah. Should it be maybe a plugin with all the systems to update some component automatically like this? I initially thought just for Inertia, bc we have one more that will be needed eventually, but the idea seemed weird to me.

Reply:
Yeah I'm not sure tbh. Probably best to keep everything together for now, and when we get the full 'essential' feature set implemented we split it into multiple plugins to make it more readable